Monika-After-Story/MonikaModDev
# Update Thumbs

Updates all the ribbon thumbs as well as some outfit thumbs to use the latest hair version.

## Testing:
- Verify that all the updated thumbs still work.
- Verify that the thumbs look good.
